include/ginkgo/core/matrix/permutation.hpp

```diff
@@ -122,21 +139,23 @@ class Permutation : public EnableLinOp<Permutation<IndexType>>,
      * @param exec  Executor associated to the LinOp
      */
     Permutation(std::shared_ptr<const Executor> exec)
-        : Permutation(std::move(exec), dim<2>{})
+        : Permutation(std::move(exec), dim<2>{}, row_permute)
```

Comment:
This change should be unneeded since you have the default properly set in the following ctor, but that's fine.